src/website/views/components/BoatCompass/BoatCompass.tsx

```diff
@@ -51,11 +52,20 @@ class BoatCompass extends React.Component<BoatCompassProps> {
     }
 
     _rotateBoat(boatAngle: number) {
-        return (boatAngle < 0) ? boatAngle + 360 : boatAngle;
+        if (boatAngle == null) {
+            return 0;
+        }
+        else {
+            return (boatAngle < 0) ? boatAngle + 360 : boatAngle;
+        }
     }
 
     _rotateBoatString(boatAngle: number) {
-        return `rotate(${this._rotateBoat(boatAngle)}deg)`;
+        if (boatAngle == null){
+            return `rotate0deg)`
+        } else {
+            return `rotate(${this._rotateBoat(boatAngle)}deg)`;
+        }
     }
```

Comment:
Question - when will `boatAngle` be `null`?

Reply:
to-do: create an issue for this after TWDR